Ticket #1528

vikasranatalentelgia opened the pull request:
Option to sort by latest topis on homepage topic list view

vikasranatalentelgia committed:
Ticket #1528

Option to sort by latest topis on homepage topic list view

akuks commented:
@vikasranatalentelgia can you please resolve the conflicts?

akuks commented:
> @vikasranatalentelgia can you please resolve the conflicts?

Can you please fix this?

vikasranatalentelgia committed:
Merge branch 'development' into talentelgia-fixes

vikasranatalentelgia commented:
@akuks conflict resolved

vikasranatalentelgia committed:
Add parameter in get all API (Sort)

Add parameter in get all API (Sort)

vikasranatalentelgia committed:
Merge branch 'development' into talentelgia-fixes

vikasranatalentelgia committed:
Ticket #797 #796

In review icon missing for camp statement
Archived camp/topic must not be editable

vikasranatalentelgia committed:
Ticket #804

Blank page when trying to support camp

vikasranatalentelgia committed:
Ticket #1530 #806

Delegate button should not be seen if a camp/topic is archived
Disable 'remove support' button for users not logged in

vikasranatalentelgia committed:
Ticket #810

When a user clicks on the camp name, the support pop-up should close.

vikasranatalentelgia committed:
Ticket #1506 #812

Duplicate API calls when user delegate support
manage support button not working
